tests/integrations/test_identity.py

```diff
@@ -0,0 +1,113 @@
+import json
+import time
+from pathlib import Path
+from typing import Generator
+
+import jwt
+import pytest
+
+from cryptography.hazmat.primitives.asymmetric import rsa
+from cryptography.hazmat.primitives import serialization
+from openeogeotrellis.integrations.identity import IDPTokenIssuer, IDP_TOKEN_ISSUER
+from openeogeotrellis.testing import gps_config_overrides
+
+
+def test_idptokenissuer_should_not_be_instantiated_manually():
+    # WHEN someone wants to manually create an instance of the IDPTokenIssuer
+    # THEN they should get an exception
+    with pytest.raises(RuntimeError):
+        IDPTokenIssuer()
+
+
+def test_without_valid_config_token_should_just_be_none(monkeypatch):
```

Comment:
`monkeypatch` is not used, so do you have to declare it?

Reply:
Nope, oversight of refactoring will fix